File: Cpp/warunek.cpp
/*
 * warunek.cpp
 * 
 */


#include <iostream>
using namespace std;

int main()
{
    int licz(0);
    int max(0);
    
    cout << "Podaj 3 liczby \n";
    for (int i(0); i<3; i++)
    {
        cin >> licz;
        if (licz > max)
        {
            max= licz;
        }
        
    }
    cout << "Największa liczba: " << max << endl;
    
	return 0;
}

File: Python/warunek.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  warunek.py
#  
def warunek():
    
    lista_liczb = []
    
    for i in range(3):
        liczba = float(input("Podaj liczbę"))
        lista_liczb.append(liczba)
    print(max(lista_liczb))


def main(args):
    
    warunek()
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
